docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,59 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SHW type
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SHW_types_and_spaces**
+- **get_SHW_equipment_connected_to_use_type**  
+
+**Applicability Checks:**
+- check that the SHW type in the U_RMD has SHW loads
+
+## Applicability Checks:
+- only projects with SHW for the SHW space type in the user model are expected to have a SHW system in the user model.  If there is no SHW in the user model, we assume there is no SHW system designed, and instead rule 11-3 Applies and P_RMD matches B_RMD.
+- use the function get_SHW_types_and_spaces to get a list of spaces for each SHW type: `shw_and_spaces_dict = get_SHW_types_and_spaces(U_RMD)`
+- look at each SHW space type: `for shw_space_type in shw_and_spaces_dict:`
+  - look at each space: `for space_id in shw_and_spaces_dict[shw_space_type]:`
+    - get the space using get_component_by_id: `space = get_component_by_id(U_RMD, space_id)`
+    - look for the ServiceWaterHeatingUse in the space.  If even one space has a ServiceWaterHeatingUse, continue to rule logic: `if len(u.space.service_water_heating_uses) > 0: CONTINUE TO RULE LOGIC`
+  - if the program reaches this line without going to the rule logic, the project is not applicable for this SHW heating use: `NOT_APPLICABLE`
+
+    ## Rule Logic: 
+    - create a boolean to keep track of whether everything matches: `all_match = TRUE`
+  - create an error string: `error_str = ""`
+  - create the compare context string: `compare_context_str = "AppG 11-1 P_RMD Equals U_RMD"`
+  - get a dictionary of all SHW equipment connected to the SHW space type in the proposed model: `p_shw_equipment_dict = get_SHW_equipment_connected_to_use_type(P_RMD)[shw_space_type]`
+  - get a dictionary of all SHW equipment connected to the SHW space type in the user model: `u_shw_equipment_dict = get_SHW_equipment_connected_to_use_type(U_RMD)[shw_space_type]`
```

Comment:
This description is a little confusing because we have the ServiceWaterHeatingEquipment object so I was not sure if you meant that or ALL the equipment. Suggest just adding what you mean in paraentheses (pumps, tasks, distribution systems, etc.).